covid_xprize/oxford_data/npi_static.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
+
+"""
+For metadata relating to NPIs -- descriptions, phases etc.
+"""
+from collections import OrderedDict
+
+NPI_COLUMNS = [
+    'C1_School closing',
+    'C2_Workplace closing',
+    'C3_Cancel public events',
+    'C4_Restrictions on gatherings',
+    'C5_Close public transport',
+    'C6_Stay at home requirements',
+    'C7_Restrictions on internal movement',
+    'C8_International travel controls',
+    'H1_Public information campaigns',
+    'H2_Testing policy',
+    'H3_Contact tracing',
+    'H6_Facial Coverings'
+]
+
+ERROR_COLUMNS = ['UpperNewCases', 'LowerNewCases']
+
+DISPLAY_COLUMNS = (['CountryName', 'Date', 'ConfirmedCases', 'NewCases',
+                   'NoNpiNewCases', 'MaxNpisNewCases', 'CurrentNpisNewCases', 'Forecast', "HistoricalNewCases"]
+                   + NPI_COLUMNS + ERROR_COLUMNS)
+
+NPI_DETAILS = [
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Strict'
+    ],
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Strict'
+    ],
+    [
+        '0: None',
+        '1: Medium',
+        '2: Strict'
+    ],
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Hard',
+        '4: Strict'
+    ],
+    [
+        '0: None',
+        '1: Medium',
+        '2: Strict'
+    ],
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Strict'
+    ],
+    [
+        '0: None',
+        '1: Medium',
+        '2: Strict'
+    ],
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Hard',
+        '4: Strict'
+    ],
+    [
+        '0: None',
+        '1: Medium',
+        '2: Strict'
+    ],
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Strict'
+    ],
+    [
+        '0: None',
+        '1: Medium',
+        '2: Strict'
+    ],
+    [
+        '0: None',
+        '1: Mild',
+        '2: Medium',
+        '3: Hard',
+        '4: Strict'
+    ]
+]
+
+# Color palette for various NPIs in the Gantt chart.
+# NPIs have different ranges (0-2, 0-3, 0-4) and this dictionary allows us to color the bars appropriately for each NPI.
+NPI_COLORS = {
```

Comment:
I don't think we use these constants in this repo. But I guess they can be useful.

Reply:
My thought was to leave in these constants just so we can match the colors we have been using. It's a little out of place, I agree, but I'd prefer this to copying them to every user interface we make.